class.two-factor-core.php

```diff
@@ -134,7 +134,7 @@ function login_html( $user, $login_nonce, $redirect_to, $error_msg = '', $login_
 		$provider = $this->get_provider_for_user( $user->ID );
 
 		$rememberme = 0;
-		if ( isset ( $_REQUEST[ 'rememberme' ] ) && $_REQUEST[ 'rememberme' ] ) {
+		if ( isset ( $_POST[ 'rememberme' ] ) && $_POST[ 'rememberme' ] ) {
```

Comment:
Leave this as $_REQUEST -- it's necessary if switching to a different provider on a branch I'm working on.